DSpace/dspace-angular
# remove authorization mocks left over from PR #1105

## References
- Removes mocks added by #1105 to mock a feature that didn't exist at the time but was added by https://github.com/DSpace/DSpace/pull/3240

## Description
#1105 added mocks for the authorization calls for certain features on certain items. From the PR description:

>Mocks:
>Since we're [waiting on the REST PR](https://github.com/DSpace/DSpace/issues/3201) to add some of the feature endpoints, I've added mocks for these endpoints on certain items. Rest api containing these items: https://api7.dspace.org/server/#/server/api
>* Authorization for `canManageBitstreams` is mocked for item 96715576-3748-4761-ad45-001646632963
>* Authorization for `canManageRelationships` is mocked for item 047556d1-3d01-4c53-bc68-0cee7ad7ed4e
>* Authorizations for `canManageVersions` and `canManageMappings` are mocked for item e98b0f27-5c19-49a0-960d-eb6ad5287067
>
>How to test:
>* Visit the edit item page for the following items:
>** 96715576-3748-4761-ad45-001646632963: The bitstreams tab should be available. Relationships, versions and collection mapper should be inaccessible. The collection mapper action on the status page should also be disabled and show a tooltip when hovered over.
>** 047556d1-3d01-4c53-bc68-0cee7ad7ed4e: The relationships tab should be available. Bitstreams, versions and collection mapper should be inaccessible. The collection mapper action on the status page should also be disabled and show a tooltip when hovered over.
>** e98b0f27-5c19-49a0-960d-eb6ad5287067: The versions and collection mapper tabs should be available. Bitstreams and relationships should be inaccessible. The collection mapper action on the status page should be active and link to the collection mapper tab.
>** Visiting any other item's edit page should have all the above actions disabled

As a consequence, if you visit any of these items in dev mode, you'd get the mocked response instead of the actual response from the server. In prod mode, the server responses are always used.

## Instructions for Reviewers
Visit the edit page for the items mentioned above using the demo backend, and verify that you have access to the same tabs as you would on the demo server, not the ones mentioned in the quoted description above.

## Checklist
